docs/core/diagnostics/cli-tools/installing.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Installing dotnet diagnostic tools - .NET Core
```

Comment:
I don't think we should add this article. Given they are simply global tools, we should simply link to https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools for more info. If there's any scenario missing, we should add to that article.